NOTICE

```diff
@@ -3,10 +3,3 @@ Copyright 2011-2016 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
-
-This product includes the following work from the Apache Thrift project:
-
-TNonblockingServerSocket.java
```

Comment:
Do we not still bundle this? Is it just that it's an ASF project and the intepretation from LEGAL-234 invalidates the need for the NOTICE entry?